pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -5,6 +5,7 @@
 , substituteAll
 , binutils
 , asciidoc
```

Comment:
```suggestion
```

See https://github.com/ccache/ccache/commit/63239e95632fb51795f0bac5f9d3c46c849e6f5d#diff-c60025a17472627516fb897916bc73880707de9aecab6b2f381c437fec7e8a88

Reply:
Ah, good point! I completely missed the fact it's a replacement. Tweaked `pkgs/top-level/all-packages.nix` and `nativeInputs` on top of your suggestion. Thank you!

Reply:
Yeah, saw it. Much appreciated!